### PR TITLE
Added user, product, and modified auth route

### PR DESCRIPTION
**Pull request by Markdiocre**

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+router.put('/update',(req, res)=>{
```
Comment (Markdiocre) [inline routes/user.js]: Panget mo magcode

File: routes/product.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    connection.end()
+})
+
+//UPDATE
+router.put('/:prodId',(req,res)=>{
+    let {prodId} = req.params
+    let {name, price} = req.body
+
+    let connection = sqlConnect()
+    let sql = `UPDATE ${table_name} SET name='${name}', price='${price}' WHERE id='${prodId}';`
```
Comment (Markdiocre) [inline routes/product.js:66]: Maybe change this to reusable scripts

File: routes/user.js (hunk 1 above)
Comment (NikoNepomuceno) [reply routes/user.js]: Sorry na po, sir

Comment (Markdiocre) [reply routes/user.js]: Ikaw rin

File: routes/product.js (hunk 2 above)
Comment (Markdiocre) [reply routes/product.js:66]: Done

**Pull request merged**